### DISCUSSION TITLE
nuc9 黑苹果谈论

### DISCUSSION THREAD

**Discussion by ccdeskde** [ccdeskde/NUC9-hackintosh#9] [General]
可以在这里谈论nuc9相关问题

**Comment by rocky930215** [comment]
大佬，之前问过雷电3的问题。暂时来说，感觉这个机器的雷电3接口有那么点问题，不论win还是mac，只要是雷电3热插拔，必会死机重启。我可以提供一个dell的雷电3拓展坞给你，之前骷髅峡谷用过一阵，现在吃灰中。

**Reply by ccdeskde** [reply]
目前除了nuc8以外我好像没见到有完美驱动雷电3接口的机器，这个等我有时间去研究下nuc8的配置文件吧

**Reply by ccdeskde** [reply]
远景论坛正在维护 等维护好了再看看

**Reply by rocky930215** [reply]
如果需要设备测试，可以发过去给你捣鼓捣鼓

**Reply by ccdeskde** [reply]
我在github上面另外一个维护nuc9efi的仓库里找到了一个不一样的ssdt文件 [ssdt](https://github.com/Xmingbai/NUC9-i5-i7-i9QNX-NUC9VXQNX-hackintosh/blob/main/NUC9-SSDT-TB3-HackinDROM.aml) 这是b站的一个up主维护的 里面的雷电三的ssdt和我这里面的不太一样，你有兴趣可以尝试更换一下

**Reply by rocky930215** [reply]
> 我在github上面另外一个维护nuc9efi的仓库里找到了一个不一样的ssdt文件 [ssdt](https://github.com/Xmingbai/NUC9-i5-i7-i9QNX-NUC9VXQNX-hackintosh/blob/main/NUC9-SSDT-TB3-HackinDROM.aml) 这是b站的一个up主维护的 里面的雷电三的ssdt和我这里面的不太一样，你有兴趣可以尝试更换一下
> 
> 

好的！我回头试一下！是直接替换、oc里启用就可以是吧？

**Reply by ccdeskde** [reply]
两种方法 第一种修改plist文件 把新的ssdt加进去 去掉老的ssdt 第二种 将新的文件命名成老的ssdt文件的名字 然后替换老文件

**Comment by deadfatty** [comment]
我在用您的EFI 5500XT 943602CS一联网没多久就会死机……系统11.3

**Comment by ccdeskde** [comment]
> 我在用您的EFI 5500XT 943602CS一联网没多久就会死机……系统11.3

换了白苹果网卡不能直接套用，自行更换白果网卡对应的kext并在oc中启用

**Reply by deadfatty** [reply]
已经启用BlueToolFixup.kext
BrcmFirmwareData.kext
BrcmPatchRAM3.kext
也定制USB把AX200的蓝牙，PCIe屏蔽了
还是不行呜呜呜

**Reply by rocky930215** [reply]
> 已经启用BlueToolFixup.kext BrcmFirmwareData.kext BrcmPatchRAM3.kext 也定制USB把AX200的蓝牙，PCIe屏蔽了 还是不行呜呜呜

不启用BlueToolFixup.kext，改用[BrcmBluetoothInjector.kext，注意注入顺序！

**Reply by ccdeskde** [reply]
正解

**Reply by deadfatty** [reply]
换了块转接卡解决啦 感谢大佬提供的EFI！OvO

**Comment by MalsonQu** [comment]
建议大佬拉个 微信群 还是qq群之类的 这样大家方便沟通 :)

**Reply by ccdeskde** [reply]
有道理，建完了，写在readme文档了面了

**Comment by marty8a8y** [comment]
大佬，请问隔空投送是否为不可用？

**Reply by ccdeskde** [reply]
增加白果网卡后可以
